### PR TITLE
About page

### PR DESCRIPTION
**Pull request by biharck**
### What
Adding about page

### Why
This way, users can have access to the repository and it is also going to help us opening issues.

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -38,6 +42,7 @@ export {
   ThumbnailEntry,
   ToolbarSection,
   Tooltip,
+  AboutPreferencesModal,
```
Comment (swederik) [inline src/components/index.js]: Why does it have the word Preferences in it? It's just an About page. There are no preferences.

File: doczrc.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ export default {
         'Study List',
         'Table List',
         'Toolbar Section',
+        'About Preferences Modal',
```
Comment (swederik) [inline doczrc.js]: Same comment as before

Comment (biharck) [reply doczrc.js]: Done

File: src/components/index.js (hunk 1 above)
Comment (biharck) [reply src/components/index.js]: Done

**Pull request merged**